Assets/Scripts/Enemy/ProjectileBehaviorScript.cs

```diff
@@ -85,6 +96,10 @@ public class ProjectileBehaviorScript : StateEntity {
         }
     }
 
+    public void projectileReflected() {
+        this.setDirection(opposites[direction]);
```

Comment:
Q; for the direction, should we flip the direction of the projectile or change it to the direction the player hit it? e.g. if the player is below a projectile traveling to the right and hits the projectile, should the projectile travel to the left (opposite direction) or to the top (direction it was hit)?